src/main/resources/db/migration/V2023.10.03.0001__add_parent_ref_to_timeline_event.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE timeline_event ADD COLUMN correlation_id UUID;
```

Comment:
I'm considering updating each existing event with a random UUID, so that I can then make the column mandatory (NOT NULL). Updating them with a "proper" correlationId, which would group them appropriately would be pretty complex and overkill for test data. Any thoughts on this?

Reply:
Ah ha .... here's the cardinality rationale 😁 (I should have waited till I'd read this comment! 🤣 )

Yes, agreed, generate and assign a random UUID for all existing timeline events (we are only in dev anyway), then make the db column non-nullable; then you can make it non-null in the DB entity and domain classes 👍